go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/turbot/steampipe-plugin-sdk/v5 v5.4.1
-	github.com/xanzy/go-gitlab v0.77.0
+	github.com/xanzy/go-gitlab v0.90.0
```

Comment:
Was this dependency upgrade required for the Events tables?
- Note to self: if including this need to check for breaking changes across all tables.

Reply:
Reverted to 0.77.0.  You can revert 2986dc8 to do the upgrade.